js/src/index.js

```diff
@@ -970,6 +974,24 @@ class JupyterScatterView extends widgets.DOMWidgetView {
     this.showLegend();
   }
 
+  zoomTargetHandler(newTarget) {
```

Comment:
I'm guessing this handler is registered via some dynamic setup, e.g.,`this[propName + "Handler"]`?

Reply:
(just didn't see anything in the diff!)

Reply:
Yep. The JS widget registers a handler for all props